sentry.client.config.ts

```diff
@@ -1,55 +1,56 @@
 // This file configures the initialization of Sentry on the client.
 // The config you add here will be used whenever a users loads a page in their browser.
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
-// import * as Sentry from "@sentry/nextjs"
-
-window.onerror = function (message, source, lineno, colno, error) {
-  console.log("Global error captured:", error)
-}
-
-window.addEventListener("unhandledrejection", function (event) {
-  console.log("Unhandled rejection captured:", event.reason)
+import * as Sentry from "@sentry/nextjs"
+
+Sentry.init({
+  enabled: ["Staging", "Sepolia", "Mainnet"].includes(process.env.NEXT_PUBLIC_SCROLL_ENVIRONMENT),
+  environment: process.env.NEXT_PUBLIC_SCROLL_ENVIRONMENT,
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  release: process.env.NEXT_PUBLIC_VERSION,
+  autoSessionTracking: false,
+  allowUrls: [/scroll\.io/, /localhost/, /vercel\.app/],
+
+  // Define how likely traces are sampled. Adjust this value in production, or use tracesSampler for greater control.
+  tracesSampleRate: 1,
+
+  // Setting this option to true will print useful information to the console while you're setting up Sentry.
+  debug: false,
+
+  // Ignore specific error messages
+  ignoreErrors: [
+    // MetaMask and other wallet plugin common errors
+    "MetaMask",
+    "Non-Error promise rejection captured",
+    "User rejected the request",
+    "User denied transaction signature",
+    // Extension injected errors
+    "Extension context invalidated",
+    "The message port closed before a response was received",
+    "Failed to fetch",
+    // RPC related errors
+    "Internal JSON-RPC error",
```

Comment:
does this mean that we need to capture errors and throw them for all interactions with the contract? 
does the bridge currently handle this?

Reply:
This configuration is intended to ignore some common, meaningless errors, mainly to reduce excess noise. It does not affect the current functionality.